fetch_resources.sh

```diff
@@ -538,12 +538,30 @@ function configure_tempest
     done
     TVAULT_IP+="]"
 
+
+    #check for test user name in TEST_IMAGE_NAME
+    #keep field empty in-case image name is other than ubuntu, centos, cirros
+    case $TEST_IMAGE_NAME in
+        *"ubuntu"*)
+                TEST_USER_NAME="ubuntu"
+                ;;
+        *"centos"*)
+                TEST_USER_NAME="centos"
+                ;;
+        *"cirros"*)
+                TEST_USER_NAME="cirros"
+                ;;
+        *)
+                TEST_USER_NAME=
```

Comment:
We should revert to default OR we should fail, if we are not able to identify test_user_name based on image name. For default, we can use "ubuntu".